packages/webpack-cli/lib/utils/cli-flags.js

```diff
@@ -1,11 +1,9 @@
 const { packageExists } = require('./package-exists');
 const cli = packageExists('webpack') ? require('webpack').cli : undefined;
 
-const HELP_GROUP = 'help';
 const BASIC_GROUP = 'basic';
 
 const groups = {
-    HELP_GROUP,
     BASIC_GROUP,
```

Comment:
Do we still need `BASIC_GROUP`?

Reply:
nope but it cant be removed directly, need some changes to basic resolver, next PR https://github.com/webpack/webpack-cli/pull/2049